app/services/fan_out_on_write_service.rb

```diff
@@ -71,6 +72,12 @@ def deliver_to_self!
     FeedManager.instance.push_to_direct(@account, @status, update: update?) if @account.local? && @status.direct_visibility?
   end
 
+  def deliver_to_thread_stream
```

Comment:
Missing `!` at the end of the method name

Reply:
Yes! Thanks for taking a look, didn't intend to commit this to this branch, still WIP, this part doesnt work yet/needs a decent bit more work :)